hashsign/jvm/src/main/scala/fluence/crypto/ecdsa/Ecdsa.scala

```diff
@@ -72,6 +73,24 @@ class Ecdsa(curveType: String, scheme: String, hasher: Option[Crypto.Hasher[Arra
         } yield keyPair
     }
 
+  def pairFromPrivate[F[_]: Monad](sk: Secret): EitherT[F, CryptoError, KeyPair] = {
+    for {
+      ecSpec ← EitherT.fromOption(
+        Option(ECNamedCurveTable.getParameterSpec(curveType)),
+        CryptoError("Parameter spec for the curve is not available.")
+      )
+      keyPair ← nonFatalHandling {
+        val hex = sk.value.toHex
+        val d = new BigInteger(hex, HEXradix)
```

Comment:
What is 'd', 'g', 'q'? Let's write full names.

Reply:
this is points on curve, there is full names :) i can add comment about this

Reply:
That looks good, thank!